projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -599,19 +596,22 @@ function get_post_access_level_for_current_post() {
 }
 
 /**
- * Renders the WP.com version of the subscriptions block.
+ * Renders the subscriptions block at the site.
  *
  * @param array $data    Array containing block view data.
  * @param array $classes Array containing the classes for different block elements.
  * @param array $styles  Array containing the styles for different block elements.
  *
  * @return string
  */
-function render_wpcom_subscribe_form( $data, $classes, $styles ) {
-	global $current_blog;
-
-	$form_id = 'subscribe-blog' . ( Jetpack_Subscriptions_Widget::$instance_count > 1 ? '-' . Jetpack_Subscriptions_Widget::$instance_count : '' );
-	$url     = defined( 'SUBSCRIBE_BLOG_URL' ) ? SUBSCRIBE_BLOG_URL : '';
+function render_for_website( $data, $classes, $styles ) {
+	$blog_id            = \Jetpack_Options::get_option( 'id' );
+	$form_id            = 'subscribe-blog-' . $data['widget_id'];
+	$form_url           = defined( 'SUBSCRIBE_BLOG_URL' ) ? SUBSCRIBE_BLOG_URL : '#';
+	$post_access_level  = get_post_access_level_for_current_post();
+	$post_id            = get_the_ID();
+	$subscribe_field_id = apply_filters( 'subscribe_field_id', 'subscribe-field-', $data['widget_id'] );
```

Comment:
```suggestion
	$subscribe_field_id = apply_filters( 'subscribe_field_id', 'subscribe-field', $data['widget_id'] );
```
I think small typo here

Reply:
I think it needs `. $data['widget_id']` added